project/frontend/src/components/ResultsMap.js

```diff
@@ -10,8 +10,8 @@ const mapStyle = {
 function MapContainer(props) {
   const [activeMarker, changeMarker] = useState({});
   const [showInfoWindow, changeInfoVisibility] = useState(false);
```

Comment:
nit: I think the more conventional names for these setter functions would be `setActiveMarker` and `setShowInfoWindow`. I think using the word `set` instead of `change` makes it more clear that these functions are setter functions that take in a parameter for the new value. I think this is mostly a matter of personal preference though, so it's up to you.

Reply:
ill change it to the convention!